common/utils.js

```diff
@@ -795,7 +795,7 @@ async function estimateITSFee(chain, destinationChain, env, eventType, gasValue,
     }
```

Comment:
Add a bit more colour to the PR title, e.g. `fix(evm): scale gas estimate for hedera`